package.json

```diff
@@ -1,44 +1,46 @@
 {
-  "name": "di-ipv-cri-toy-api",
-  "version": "1.0.0",
+  "name": "ipv-cri-toy-api",
   "description": "API for the toy credential issuer",
   "main": "index.js",
   "scripts": {
+    "pre-commit": "pre-commit run --all-files --show-diff-on-failure",
     "lint:prettier": "prettier --check lambdas",
     "lint:prettier-fix": "npx prettier --write .",
     "lint:eslint": "eslint .",
-    "lint": "npm run lint:prettier && npm run lint:eslint",
+    "lint": "npm run lint:eslint",
+    "lint:fix": "npm run lint:eslint -- --fix",
```

Comment:
Can we replace this with a [eslint pre-commit plugin](https://github.com/pre-commit/mirrors-eslint)?

Reply:
Added eslint plugin to pre-commit